hipaa/static/hipaa/ping.js

```diff
@@ -53,7 +53,16 @@ $(document).ready(function(){
                 location.reload(true);
             }
             state = response.state
-            if(response.seconds_until_next_ping <= 30 && $('#hipaa-ping-warning').length == 0){
+            if(response.seconds_until_next_ping <= response.seconds_before_logout_to_display_warning && $('#hipaa-ping-warning').length == 0){
+                // Check if a video was playing and pause it.
+                var was_playing;
+                $("video").each(function(){
+                    if(!$(this).get(0).paused){
+                        was_playing = $(this).get(0)
```

Comment:
You'll want to select audio tags too.

This might be overkill, but if there was more than one audio/video element playing, this code would pause all the playing ones, but when the user clicked the warning banner, only the *last* video/audio element would start playing again. If you wanted it to be more robust, you could make was_playing an array of `<audio>` and `<video>` elements, and then loop through them in `div.click` and call `play()` on them. (But again, it is very unlikely that multiple video or audio elements will ever be playing simultaneously on an application that uses django-hipaa)